packages/auth/src/authClients/netlify.ts

```diff
@@ -47,5 +47,8 @@ export const netlify = (client: NetlifyIdentity): AuthClient => {
     getUserMetadata: async () => {
       return client.currentUser()
     },
+    restoreAuthState: async () => {
+      return client.currentUser()
+    },
```

Comment:
Could you leave a comment for what this is/ does or when it's meant to be used?

Reply:
// restore authentication when an OAuth or magiclink callback
    // redirects back to site with access token